message_definitions/v1.0/messages.xml

```diff
@@ -1908,27 +1928,27 @@
-      <field name="angular_accel_r" type="float" unit="rad/s2"/>
-      <field name="angular_accel_ref_p" type="float" unit="rad/s2"/>
-      <field name="angular_accel_ref_q" type="float" unit="rad/s2"/>
-      <field name="angular_accel_ref_r" type="float" unit="rad/s2"/>
-      <field name="g1_p" type="float"/><!-- INDI Simple G-matrix -->
-      <field name="g1_q" type="float"/>
-      <field name="g1_r" type="float"/>
-      <field name="g2_r" type="float"/>
-      <field name="u" type="float[]"/><!-- outputs -->
+    <message name="STAB_ATTITUDE" id="216">
+      <field name="phi"                 type="float"    unit="rad"     alt_unit="deg"      alt_unit_coef="57.29578"/><!-- attitude tracking -->
+      <field name="theta"               type="float"    unit="rad"     alt_unit="deg"      alt_unit_coef="57.29578"/>
+      <field name="psi"                 type="float"    unit="rad"     alt_unit="deg"      alt_unit_coef="57.29578"/>
+      <field name="phi_ref"             type="float"    unit="rad"     alt_unit="deg"      alt_unit_coef="57.29578"/>
+      <field name="theta_ref"           type="float"    unit="rad"     alt_unit="deg"      alt_unit_coef="57.29578"/>
+      <field name="psi_ref"             type="float"    unit="rad"     alt_unit="deg"      alt_unit_coef="57.29578"/>
+      <field name="angular_rate_p"      type="float"    unit="rad/s"   alt_unit="deg/s"    alt_unit_coef="57.29578"/><!-- rate tracking -->
+      <field name="angular_rate_q"      type="float"    unit="rad/s"   alt_unit="deg/s"    alt_unit_coef="57.29578"/>
+      <field name="angular_rate_r"      type="float"    unit="rad/s"   alt_unit="deg/s"    alt_unit_coef="57.29578"/>
+      <field name="angular_rate_ref_p"  type="float"    unit="rad/s"   alt_unit="deg/s"    alt_unit_coef="57.29578"/>
+      <field name="angular_rate_ref_q"  type="float"    unit="rad/s"   alt_unit="deg/s"    alt_unit_coef="57.29578"/>
+      <field name="angular_rate_ref_r"  type="float"    unit="rad/s"   alt_unit="deg/s"    alt_unit_coef="57.29578"/>
+      <field name="angular_accel_p"     type="float"    unit="rad/s^2" alt_unit="deg/s^2"  alt_unit_coef="57.29578"/><!-- angular acceleration -->
+      <field name="angular_accel_q"     type="float"    unit="rad/s^2" alt_unit="deg/s^2"  alt_unit_coef="57.29578"/>
+      <field name="angular_accel_r"     type="float"    unit="rad/s^2" alt_unit="deg/s^2"  alt_unit_coef="57.29578"/>
+      <field name="angular_accel_ref_p" type="float"    unit="rad/s^2" alt_unit="deg/s^2"  alt_unit_coef="57.29578"/>
+      <field name="angular_accel_ref_q" type="float"    unit="rad/s^2" alt_unit="deg/s^2"  alt_unit_coef="57.29578"/>
+      <field name="angular_accel_ref_r" type="float"    unit="rad/s^2" alt_unit="deg/s^2"  alt_unit_coef="57.29578"/>
+      <field name="nu"                  type="float[]"  unit="none"/><!-- inputs-->
+      <field name="u"                   type="float[]"  unit="none"/><!-- outputs -->
```

Comment:
These names are not very clear. Maybe use `inputs` and `outputs` ? You can also make a better description of some the fields with `<field ...>description goes here</field>`. It will be integrated to the generated documentation which is not the case of XML comments.

Reply:
I have added a description